tests/defaultFetchAssoc/config/bootstrap.php

```diff
@@ -21,12 +21,13 @@
 $config = RuntimeConfiguration::create();
 $config->errorMode(RuntimeConfiguration::ERROR_MODE_EXCEPTION);
 $config->defaultFetchMode(QueryReflector::FETCH_TYPE_ASSOC);
+// $config->debugMode(true);
 
 try {
     if (false !== getenv('GITHUB_ACTION')) {
         $mysqli = @new mysqli('127.0.0.1', 'root', 'root', 'phpstan_dba');
     } else {
-        $mysqli = new mysqli('localhost', 'testuser', 'test', 'phpstan_dba');
+        $mysqli = new mysqli('mysql80.ab', 'testuser', 'test', 'phpstan_dba');
```

Comment:
just realized that you might have different local access parameters in your dev setup @xPaw 

created a issue to support this use-case better:  https://github.com/staabm/phpstan-dba/issues/267